stories/example-screen-form-view.stories.js

```diff
@@ -0,0 +1,451 @@
+            </div>
+          </div>
+          <div class="d-flex justify-content-right">
+            <span class="fs-6 fw-light" role="button" tabindex="0">Edit</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+`
+
+const tabsNavigationHtml = `
+  <div class="w-100 overflow-auto p-0 mb-0 border border-light">
+    <div class="bg-white rounded-top border border-light d-flex">
+      <button class="d-flex align-items-center bg-white border-0 text-light rounded-top">
+        <svg class="icon-lg-font-size">
+          <use xlink:href="${iconSprite}#tabler-chevron-left" />
+          <span class="visually-hidden">Extend tabs' options</span>
+        </svg>
+      </button>
+      <ul class="w-100 overflow-hidden flex-nowrap nav nav-tabs nav-tabs-single-row-with-controls"></ul>
+      <button class="d-flex align-items-center bg-white border-0 text-warning rounded-top">
+        <svg class="icon-lg-font-size">
+          <use xlink:href="${iconSprite}#tabler-chevron-right" />
+          <span class="visually-hidden">Extend tabs' options</span>
+        </svg>
+      </button>
+    </div>
+  </div>
+`
+
+const form_view_html = `
+  <div class="m-0 bg-light">
+    <div class="mx-auto my-0">
+      ${LayoutLoggedHeader()}
+      <div class="d-flex mt-4 w-100">
+        <div class="d-none d-lg-block">
+          ${LayoutSidebar()}
+        </div>      
+        <div class="d-flex flex-column justify-content-start flex-wrap w-100 px-4">
+          <div class="d-flex justify-content-between align-items-center flex-wrap px-3 w-100">
+            <h2 class="m-0">00012-001-2020 (Billy Goatgruff - A1234421)</h2> 
+            <button type="button" class="btn btn-danger fs-6">
+              <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
+                <use xlink:href="${iconSprite}#tabler-chevron-down" />
+              </svg>Other Actions
+            </button>
+          </div>
+          <div class="bg-white my-3 w-100 rounded-4">
+            ${tabsNavigationHtml}
+            <div class="row g-0">
+              <div class="col-md-8 col-sm-12 px-9 py-4">
+                <span class="text-warning fw-bold">Incident Info</span>
+                <div class="w-100 fs-5 py-4 incident-report-list"></div>
+                <div>
+                  <button type="button" class="btn btn-outline-secondary fs-5 me-2">
+                    <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
+                      <use xlink:href="${iconSprite}#tabler-pencil" />
+                    </svg>Edit
+                  </button>
+                  <button type="button" class="btn btn-outline-secondary fs-5">
+                    <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
+                      <use xlink:href="${iconSprite}#tabler-letter-x" />
+                    </svg>Cancel
+                  </button>
+                </div>
+              </div>
+              <div class="col-md-4 col-sm-12 px-9 py-4">
```

Comment:
Let's change this, so that we have 20px padding around the container:

`<div class="col-md-4 col-sm-12 p-7">`

broken:
![2021-09-08_08-37-42AM](https://user-images.githubusercontent.com/1605754/132519973-5fa2ec9f-639f-4c7c-8641-b2a8691cb59b.png)

fixed:
![2021-09-08_08-37-54AM](https://user-images.githubusercontent.com/1605754/132519990-2345ec64-2b4c-414c-a8d2-69e91188739b.png)